.github/workflows/test-all.yml

```diff
@@ -84,6 +84,336 @@ jobs:
           cp -r packages/react-native/sdks/hermes/* $HERMES_WS_DIR/hermes/.
 
           echo ${{ steps.hermes-version.outputs.version }}
+  build_hermesc_apple:
+    runs-on: macos-13
```

Comment:
what's the size of this machine? We can go to large and xlarge if we need it. :D

Reply:
This machine has 4 CPUs, 14G RAM, 14G SSD. The larger macos runners are described [here](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#about-macos-larger-runners). You could certainly go bigger is so desired.